engine/lib/phx/src/input/input.rs

```diff
@@ -7,6 +7,7 @@ use crate::*;
 use crate::system::*;
 
 use sdl2_sys::*;
+use tracing::warn;
```

Comment:
Bit of a nitpick, but could we perhaps reexport these from the logging module? It feels part of that module, but in this case we happen to use an external library.

Reply:
as an aside, can `Fatal` become `fatal`? :P